### DISCUSSION TITLE
Welcome to transformer-time-series-prediction Discussions!

### DISCUSSION THREAD

**Discussion by oliverguhr** [oliverguhr/transformer-time-series-prediction#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by falihgoz** [comment]
Hi,

I love your work! I am Falih. I am currently a master student. I am trying to understand your code and just want to ask you a question. Could you please tell me why did you put feature_size = 250 instead of 100 which is also input_window?

**Reply by oliverguhr** [reply]
Hi Falih, 
one point in time will be represented by one feature vector. I did set the length of the feature vector to 250 because it worked for me. All you need to take care of is, that the length of the feature vector is divisible by the number of heads. You can probably get good results with a feature vector length of 100 or less, depending on your data.

**Comment by kumar598** [comment]
hi, how can i find accuracy ?
